diffkemp/simpll/ModuleComparator.cpp

```diff
@@ -203,13 +217,15 @@ void ModuleComparator::compareFunctions(Function *FirstFun,
             DEBUG_WITH_TYPE(DEBUG_SIMPLL, decreaseDebugIndentLevel());
             if (result == 0) {
                 DEBUG_WITH_TYPE(DEBUG_SIMPLL,
-                                dbgs() << getDebugIndent() << "After inlining, "
-                                       << "the functions are equal\n");
+                                dbgs() << getDebugIndent()
+                                       << Color::makeGreen("equal")
+                                       << " after inlining\n");
                 ComparedFuns.at({FirstFun, SecondFun}).kind = Result::EQUAL;
             } else {
                 DEBUG_WITH_TYPE(DEBUG_SIMPLL,
-                                dbgs() << getDebugIndent() << "After inlining, "
-                                       << "the functions are not equal\n");
+                                dbgs() << getDebugIndent() << "still "
+                                       << Color::makeRed("not equal")
+                                       << " after inlining\n");
```

Comment:
Sometimes, this can be quite far from the original function comparison. Could you add function names so that it's clearer? Something like:
```
"function1" and "function2" still not equal after inlining
```
with `not equal` being red.